tox.ini

```diff
@@ -2,11 +2,11 @@
 envlist = py{27,32,33,34},
           pypy
           pypy3
-
+# Note that py32 depends on virtualenv < 14.0.0
```

Comment:
I don't think it makes sense to support py32 anymore. Need for support for py33 is also questionable. We can probably simplify a bit our test suite by getting rid of them